### ISSUE TITLE
Create a checklist/guidelines for accessible deliverables

### ISSUE THREAD

**Issue by lauranash18f** [18F/TLC-crew#87]
### Description
Create a checklist or guidelines for ensuring accessible "everyday" deliverables — slides/presentation tips, documents, tables, charts, and diagrams. This work would help with onboarding, consistency, and practicing accessibility every day. It would also help us model our commitment to accessibility outside of software design/development.

**Why now?**

- **Building on recent training**: Many team members just completed WebAIM accessibility training, and can apply the information learned during this outside of web contexts.
- **Creating a solid foundation**: As people use delivery assurance time to develop internal training and materials, this is a way to ensure that work is accessible from the start.

### Point of contact on this issue
Posed by Laura Nash, seconded by Melissa Braxton and others in the WebAIM accessibility training. Anyone in the training (#webaim-accessibilty-training-march-2023) should be able to assist with questions; #g-accessibility may also be able to help.

**Billable?**

- [ ] Yes
- [x] No
If yes, tock code:

**Skills needed**

- [x] Any human
- [ ] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [x] No

**How much time do you anticipate this work taking?**
Depends on scope and knowledge of the creator; minimum list could be a day or two but more extensive advice across multiple categories could take a whole sprint. I could see this becoming multiple tickets as well, if contributors are comfortable with some types of deliverable but not others.

### Acceptance Criteria

- [x] What does "done" look like?
As an 18F staff member who is preparing a new slide deck, I can find out what actions to take to ensure both the file and the presentation itself will be accessible.

As an 18F staff member who is creating a new document (text, table, chart, or diagram), I find out what actions to take to ensure it is accessible.

Note: I'd also want this to be part of the resource library and shared in #g-accessibility.

### Tasks
 - [x] Document guidance
 - [x] Add to project resources folder
 - [x] Add link to product resources library
 - [x] Share in the accessibility channel

**Comment by lauranash18f** [comment]
Anne Petersen mentioned that digital.gov has some of this material as well, which we can build on.

**Comment by egamble23** [comment]
@lauranash18f Hi Laura! I'm working on this ticket. Just to check, would the main tools used to create these deliverables (slides, docs, etc.) be Google-based? Or would there be also a need for Microsoft-based tools and creating accessible deliverables using those?

Each platform may handle accessibility a little differently so just want to clarify that as I'm doing desk research and drafting an outline. Thanks!

**Comment by egamble23** [comment]
Thanks so much, @lauranash18f , for the clarifications! I've been working through the desk research I've compiled and have made progress on an outline for the checklist. Based on the scope described here, I'll focus on Google-based tools for now. Will follow up as more progress is made! Thanks again!

**Comment by egamble23** [comment]
Hi @lauranash18f , a quick question in case you're aware of it - is there existing guidance around accessible fonts/typography for Google-based deliverables already?

I've been looking around and found on USDWS some information for website font styling (https://designsystem.digital.gov/utilities/text-styles/) and 18F on color contrast for text (https://accessibility.18f.gov/checklist/). However, I may be overlooking something specific to deliverable PPT/slides, documents, etc. (e.g., don't have text in your PPT be below this font size; we use only these colors, etc.).

Note, I will be including in the checklist a link to the 18F Visual Identity > Templates > Google Docs: https://brand.18f.gov/templates/, but it doesn't get into the details about min. font sizing, etc.  

This guidance may not exist yet, which is no problem at all! :) Just wanted to check! 

Thanks in advance for the clarifications! I can also ask this in the Accessibility Guild Slack channel, if that's a better fit for this question?

**Comment by egamble23** [comment]
@lauranash18f -- Hi Laura, here is a draft guidelines/checklist doc. Very open to feedback/refinements! Just intended to be a starting point. :)

A few notes:
- The top notes will go away once it's finalized! It's just a tracking section for now :)

- To avoid repeating the same guidance for each tool (e.g., not listing alt text guidelines 4-5 times under a tool heading), I started with overarching guidelines and then followed with additional checklist items tailored to each tool. Hoping it's easier to read/use that way -- warmly welcome feedback, though!

- I referenced other resources at the end; these may be streamlined in the final iteration to just a few ones

- Fyi, while it doesn't seem Google offers an automated checking tool like Microsoft does, there are some add-ons available including some universities using Grackle (see comment for more info). Not saying it's right for us, more just something I found during research

For Laura's input [https://docs.google.com/document/d/1YhEKKT9KDXkZXFjCGQpk9V0Bv-_biUgakEV9Q8g_UXs/edit?usp=sharing]

After your feedback, I can refine and then circulate with the Accessibility Guild for more input as well as others. Warmly welcome feedback/refinements/comments!

**Comment by lauranash18f** [comment]
Added a few notes in the doc, happy to talk more about it! Very happy this is happening : )

**Comment by egamble23** [comment]
This feedback is so helpful, @lauranash18f ! Really appreciate these thoughtful notes and feedback!

Let me refine a bit further based on your feedback. I'll also look to set up a brief touching base to discuss the refinements and next steps for further input/review/etc. Thank you again! Making progress :)

**Comment by egamble23** [comment]
@lauranash18f , just a heads up, I've shared the draft checklist in the Accessibility Guild and the recent WebAIM training channel. I'll refine and iterate based on feedback. Will follow up. :)

**Comment by egamble23** [comment]
Just to keep this thread updated, @evoeges will kindly to a final accessibility review of the Google-based tool checklist later this week. Many thanks, Elyse!

**Comment by evoeges** [comment]
I did a first pass review of the draft! I'd love to spend a little more time on a more detailed review but I'll be OOO tomorrow (Friday). If there's no urgency on this ticket, I'll pick it back up Monday. Otherwise I hand it off to you @egamble23.

**Comment by egamble23** [comment]
Big, big thanks, @evoeges !! This is such valuable feedback! I've accepted/woven in your refinements. I made a few tweaks in language to align with your suggestions. 

I'm not sure it's a time sensitive issue, so unless others think differently, I think it's fine to have another take on Monday, no worries. :) Have a good Friday and thanks again for your fresh look at this! So helpful!

**Comment by MelissaBraxton** [comment]
Great! Please add this to project resources folder. 

@lauranash18f  - We might also want to add a link in the Airtable resources inventory? Any other places you think we'll want to link to this from? 

@egamble23  - Also think about whether we want to announce the existence of this new resource in Slack. #18f might be good.

**Comment by lauranash18f** [comment]
I think it would be suitable to add to the onboarding checklist, and also
to make sure the Accessibility Guild knows about it so they can add it to
their resource folder/pinned links in their Slack channel.

So glad it's here!

**Comment by egamble23** [comment]
Thanks, @MelissaBraxton ! Super helpful! I've added a clean version of the file to the [Project Resources folder here](https://docs.google.com/document/d/1aQfHNGAwwOaBGllGeZ9_xA5aeYd3Ak6TQL4-M8B5SVg/edit?usp=sharing) and I've added it to the Project Resources Airtable. I will post in the Accessibility Guild and 18F Slack channels today. 

@lauranash18f , thanks for your comment! Sounds good about also suggesting it be added to the onboarding checklist. I'm still pretty new -- is there a particular person I should reach out to with that suggestion? Or would it be someone in People Ops? 

Warmly welcome tips! Thanks!
